lib/sign-in-component.js

```diff
@@ -14,6 +14,9 @@ class SignInComponent {
     this.disposables.add(this.props.commandRegistry.add(this.element, {
       'core:confirm': this.signIn.bind(this)
     }))
+    this.props.workspace.observeTextEditors(editor => {
+      this.attachChangeEvent()
```

Comment:
It is needed there so that way when the dialog first comes up, the editor is wired up to the event. If this code is commented out, then the button will never be able to change state. I have a test case that demonstrates this. If there is a more appropriate way to handle this, I'm all ears and will address it ASAP!

Reply:
I’m still new to using etch how so I wasn’t aware of when exactly the refs were available. Learn something new every day!

Reply:
You and me both! I just learned this today, too. 😸